package.json

```diff
@@ -5,12 +5,13 @@
   "dependencies": {
     "@emotion/react": "^11.10.5",
     "@emotion/styled": "^11.10.5",
-    "@mui/icons-material": "^5.10.16",
+    "@mui/icons-material": "^5.14.16",
     "@mui/material": "^5.10.17",
     "axios": "^1.2.1",
     "next": "^13.0.6",
     "react": "^18.2.0",
-    "react-dom": "^18.2.0"
+    "react-dom": "^18.2.0",
+    "styled-components": "^6.1.0"
```

Comment:
The reason why I added styled-components is that mui is limited to customization. My implementation of styled components is independent and doesn't touch mui, so it can still live together. Also, I personally think it's more readable. 
Yet, if this change is still needed, I'd like to open a new PR.

Reply:
That's fair, although you can use emotion in place of styled-components without using MUI

```jsx
import styled from "@emotion/styled";

const StyledComponent = styled.div`
  ...
`
```

Reply:
Added in https://github.com/shybyeapp/shybye-web/issues/31